context-py/src/lib.rs

```diff
@@ -20,9 +20,13 @@ define_stub_info_gatherer!(stub_info);
 
 #[gen_stub_pyfunction]
 #[pyfunction]
-fn env_parse(env_vars: HashMap<String, String>) -> Option<env::parser::CIInfo> {
+#[pyo3(signature = (env_vars, stable_branches))]
```

Comment:
I believe this should fix the trailing optional warning: 
```
use of deprecated constant `__pyfunction_env_parse::SIGNATURE`: this function has implicit defaults for the trailing `Option<T>`
```
ref: https://pyo3.rs/v0.23.3/function/signature.html